scripts/joinmarket-qt.py

```diff
@@ -2332,6 +2335,12 @@ def get_wallet_printout(wallet_service):
                                     entry.serialize_amounts(),
                                     entry.serialize_status(),
                                     entry.serialize_label()])
+        # Append the account xpub to the end of the list                                    
```

Comment:
Please remove the extra whitespace at the end of the line.

(tip: before you push do a `git diff` on the command line and it'll show unwanted whitespace in red).